### DISCUSSION TITLE
Does not log the contract address

### DISCUSSION THREAD

**Discussion by RyanKoech** [reach-sh/reach-lang#804] [Q&A] [answered]
https://github.com/reach-sh/reach-lang/blob/21b070248feb0a345f8ceb1a6ea180853a617103/examples/rps-8-interact/index.mjs#L32

Why does my code output ```The contract is deployed as = {"type":"BigNumber","hex":"0x50"}``` then stall?

**Comment by jeapostrophe** [comment, marked as answer]
The instructions say to run two terminals --- https://docs.reach.sh/tut/rps/#p_327

You have just run the program as Alice. Now you need to run the program as Bob. The two instances of the program will talk to each via the blockchain network.

**Reply by RyanKoech** [reply]
Thank you for your reply, but I don't necessarily agree because I tried it on multiple terminus, and it was still brings up the same issue. Unless there is somewhere else I am going wrong or probably some configurations.

**Reply by jeapostrophe** [reply]
What exactly did you type in to the other terminals? You could paste here your entire interaction with the program

**Reply by RyanKoech** [reply]
I have realized and now believe there is not really an issue. The output was correct. I was just expecting it to be the hexadecimal value as in the tutorial. https://docs.reach.sh/tut/rps/#p_330. Isn't it?

**Reply by jeapostrophe** [reply]
Every connector has different formats. The Web page just shows it for Ethereum.

**Reply by RyanKoech** [reply]
Thanks once more 😅!
